### PR TITLE
Write cpuid to /boot/cpuid.txt

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR:
 * Changes kano-network-hook to save the cpuid to /boot/cpuid.txt as well as /etc/
   We don't put the cpuid in the filename in /boot to avoid confusing  users. Also don't write it more than once.

 * Fixes return value not to cause systemd to think the process has failed if
the cpuid was already sent.

For https://trello.com/c/9zIfnzFG

### CODE REVIEW

File: bin/kano-network-hook [hunk 1]
```diff
@@ -29,9 +30,14 @@ def send_cpu_id():
...
-    cpuid_filename = '/etc/cpuid-{}'.format(get_cpu_id())
+    cpuid = get_cpu_id()
+    cpuid_filename = '/etc/cpuid-{}'.format(cpuid)
+    cpuid_filename_boot = '/boot/cpuid.txt'.format(cpuid)
+    utc_time = time.asctime(time.gmtime(time.time()))
+
     if os.path.isfile(cpuid_filename):
         # We have already sent the CPU ID
+        sent = True
         pass
```
Comment (skarbat) [inline bin/kano-network-hook]: `pass` not needed anymore?

File: bin/kano-network-hook [hunk 2]
```diff
@@ -40,21 +46,33 @@ def send_cpu_id():
...
         except:
             if os.path.isfile(cpuid_filename): os.unlink(cpuid_filename)
             logger.error('Error sending Kit serial number')
- 
+
+    if os.path.isfile(cpuid_filename_boot):
+        pass
+    else:
+        try:
+            with open(cpuid_filename_boot, 'a') as f:
```
Comment (skarbat) [inline bin/kano-network-hook]: probably innocuous, but should'nt it be accessed in `w`rite instead of `a`ppend?

File: bin/kano-network-hook (hunk 1 above)
Comment (Ealdwulf) [reply bin/kano-network-hook]: Yes, I'll remove.

File: bin/kano-network-hook (hunk 2 above)
Comment (Ealdwulf) [reply bin/kano-network-hook]: True.

**Pull request merged**